aasemble/django/apps/api/tests.py

```diff
@@ -146,6 +146,18 @@ def test_delete_repository_invalid_token(self):
         response = self.client.delete(repo['self'])
         self.assertEquals(response.status_code, 401)
 
+    def test_delete_repository_new_user(self):
```

Comment:
Maybe rename it to `test_delete_repository_other_user`. The fact that Aaron is a new user isn't the important part. It's important that it's not his repository.